idflakies-core/src/main/java/edu/illinois/cs/dt/tools/utility/PathManager.java

```diff
@@ -0,0 +1,63 @@
+package edu.illinois.cs.dt.tools.utility;
+
+import com.google.common.base.Preconditions;
+import edu.illinois.cs.dt.tools.utility.Level;
+import edu.illinois.cs.dt.tools.utility.Logger;
+import edu.illinois.cs.testrunner.configuration.Configuration;
+import org.apache.maven.project.MavenProject;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class PathManager {
+    private static final String outputPath = System.getProperty("dt.cache.absolute.path", "");
+
+    public static Path modulePath(final File baseDir) {
+        return baseDir.toPath();
+    }
+
+    private static MavenProject getMavenProjectParent(final MavenProject mavenProject) {
+        MavenProject parentProj = mavenProject;
+        while (parentProj.getParent() != null && parentProj.getParent().getBasedir() != null) {
```

Comment:
Maybe we should add an additional condition at the beginning `parentProj != null`, just in case the starting `mavenProject` is already null for some reason.

Reply:
If this is the case, are we simply returning null?

Reply:
Yes.